packages/utils/src/node/authorisation/decode.token.ts

```diff
@@ -0,0 +1,10 @@
+import jwt from 'jsonwebtoken'
+
+export const decodeToken = (
```

Comment:
Like I say, I don't think we need this func - we should validate tokens too and the existing one does this